src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -137,7 +137,7 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
                                             isInline
                                         >
                                             {complianceFetchStatus === 'fulfilled' && Array.isArray(complianceSummary.data) &&
-                                                3 - complianceSummary.data.length >= 1 &&
+                                                complianceSummary.data.length - 3 >= 1 &&
                                                     `${3 - complianceSummary.data.length} more compliance reports`
```

Comment:
What is `3` anyway (besides a magic number)? We should define a descriptive constant.

Reply:
Right now the 3 is the max amount of policies displayed in the dashboard - we don't know how much longer we're going to keep this dashboard though

Reply:
This question wasn't because I didn't understand what it was but more to say: please define a constant, but if you're good with the magic number, then we can keep it.